superset/contrib/docker/docker-entrypoint.sh

```diff
@@ -17,6 +17,15 @@
 #
 set -ex
 
+if [ "`whoami`" = "root" ] && [ -n "$LOCAL_USER" ]; then
```

Comment:
could you please add comment why you do it here

Reply:
If the user is `root` ([as explained below](https://github.com/src-d/sourced-ui/pull/221#discussion_r307387875)), internal `superset` user `UID` is changed to match the user running docker `$LOCAL_USER`.
As explained in the PR desc, it is done because that way all the data written by `superset` user, will be also accessible through the volumes, by the local user, outside of the container.

Reply:
I mean add comment into code

Reply:
The problem, at least in Linux, is that the data written in a volume is owned by the user writing it, so if it is written by the local user, it won't be accessible for internal `superset` user; and the opposite: data written by internal `superset` user, won't be accessible for the local user out of the container.

This problem is avoided if both users: internal and external ones, share the same `UID`

Reply:
In this case, comment would be something like `change permissions for files coming from image to allow to write data on host file system` or something else if I understood it wrong.
Just let's make it short.

Reply:
would this comment be enough in your opinion @smacker ?
>_If the user running the shell is `root`, and it was provided the `UID` of the user running the host machine, it will be assigned that `UID` to the internal `superset` user, and then, the `/entrypoint.sh` will be ran as `superset` user (see more details at https://github.com/src-d/sourced-ui/pull/221)_